src/services/images.js

```diff
@@ -0,0 +1,9 @@
+const baseUrl = "https://image.tmdb.org/t/p/w220_and_h330_face";
+
+// get imageId from movie and return image
+const getImages = (imageId) => {
+  const response = baseUrl + imageId;
+
+  return response;
+};
+export default getImages;
```

Comment:
you can make a helper.js for this kind of operations, services serve for API request handling and staff like that